integreat_cms/static/src/js/feedback/toggle-feedback-comments.ts

```diff
@@ -1,7 +1,10 @@
 /*
  * This file provides a simple spoiler for feedback comments with hide/show functionality.
  */
+import feather from "feather-icons";
+
 window.addEventListener('load', () => {
+    feather.replace({ class: 'inline-block' });
```

Comment:
Since the icons are also replaced in the index.ts, this will cause the icons to be replaced exactly twice on load:
https://github.com/digitalfabrik/integreat-cms/blob/25b2b7b54d942fa3f58fa7fc5ef6ee8387d45e34/integreat_cms/static/src/index.ts#L74-L76
Maybe it would be better to reduce this, e.g. by exporting the feedback event handler function and call it in index.ts after the icons have been replaced or set the event handler on the parent `<span>` element?